REVIEWING.md

```diff
@@ -0,0 +1,111 @@
+Disagreements between maintainers can generally be worked out during PR review through comments.
+If this is not possible, other maintainers can be called on to give their opinions and determine a way forward.
+We encourage such disagreements and discussions, so long as they remain respectful.
+
+Being respectful includes respect for the contributor's time.
+If there is a disagreement, please do not make the contributor make repeated changes until an agreement on how to proceed is reached.
+Also, please attempt to reach an agreement quickly, so the PR can be merged in a timely fashion.
+
+### Language Version Updates
+
+Most repositories in the Podman project are written in Go, and as such target a specific version of Go in their `go.mod`.
+For example:
+```
+$ cat go.mod | grep 'go 1.'
+go 1.22.8
+```
+
+For all branches except the main branch, this should always remain static.
+PRs into a non-main branch which change the supported Go version should be modified to not require such a change, or rejected if that is not possible.
+Changing supported Go version in the main branch is allowed, but not encouraged.
+Doing so will require significant discussion among maintainers, as it can effect on what distributions the main branch can be built.
+
+### Tests
+
+Please Check tests added by the PR.
+If a PR has no new tests, determine if this is actually appropriate - has new functionality been added which should have been tested?
+If the PR does have tests, check to see if they are reasonably comprehensive.
+The Podman Project does not have code coverage standards at present, but we aim to ensure that all new functionality is tested.
+
+### Documentation
+
+All changes to public-facing APIs (e.g. the Podman REST API) and CLI should be appropriately documented.
+API changes require Swagger documentation.
+CLI changes should be documented in the Manpages.
+
+Please validate that PRs making such changes include appropriate documentation.
+Many repositories in the Podman project will enforce this via CI check, but you should still review the contents of the documentation to make sure they are appropriate and complete.
+
+## Things to Avoid
+
+### Bikeshedding and excessively critical reviews
+
+Please avoid bikeshedding during reviews.
+
+Trivial changes that do not effect the ultimate functionality of the PR - for example, small grammer or phrasing changes in documentation, unnecessary renaming of variables - should not block merge of a PR.
+It is acceptable to make such comments, but they should be marked as nice to have changes.
+
+Also, it should not have to be said, but please be polite during code reviews.
+The [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) applies to all reviewers, of course, but in order to encourage repeat contributions, reviewers should be polite and pleasant when performing reviews.
```

Comment:
Politeness can be hard to judge given the same language is spoken differently around the world. Can we add `assume good intent` here? Also, if there's a recommended list of words to avoid that can be linked here, that'd be nice I guess.

Reply:
Sure